### PR TITLE
Add/update docs for new 1.15 features

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

**Review by vapurrmaid — CHANGES_REQUESTED** [review]
## Requested Changes

- Cluster Info screenshot shows "Edit", but that's internal-only right now as the feature is under development. I'd disable the feature flag and re-take the shot

- PWA page will need a further overhaul. We have removed the _root_ PWA (ie the majority of the dashboard), opting instead for a PWA-per-application approach.

**Comment by khorne3** [comment]
@vapurrmaid 

- Updated Cluster Info screenshot to exclude the edit button
- Changed the organization of the PWA doc so that legacy info is still available for those on older versions of Coder

File: environments/pwa.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
 ---
 title: "Progressive Web Apps"
-description: Learn how to install the editor PWAs for a browser editing experience closer to native.
+description: Learn how to install the editor PWAs
+for a browser editing experience closer to native.
```
Comment (vapurrmaid) [inline environments/pwa.md:4]:
```suggestion
for a browser editing experience closer to native IDEs.
```

File: environments/pwa.md [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
 ---
 
-In addition to running Coder in a web browser, you can also run it as
-a progressive web application (PWA). Using Coder as a PWA offers you
-an experience that is akin to a native application and offers improved
-performance.
+In addition to running Coder in a web browser, you can also run it as a
```
Comment (vapurrmaid) [inline environments/pwa.md:7]:
```suggestion
In addition to running Coder in a web browser, you can also run each application as a
```

Probably will fail lint as-is due to line-length, but just to demonstrate wording

File: environments/pwa.md [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
-To use Coder as a PWA, you must be using Google Chrome or Microsoft
-Edge.
+To use Coder as a PWA, you must be using Google Chrome or Microsoft Edge.
 
-> You must be logged in to Coder using either Chrome or Edge before
-> proceeding with any of the following installation steps.
+## Installing the PWA
+
+1. Log into Coder and select your environment.
+2. Under **Applications** click **code** to launch code-server.
```
Comment (vapurrmaid) [inline environments/pwa.md:18]:
```suggestion
2. Under **Applications**, click on an application in order to launch it as a new window (or tab)
```

File: environments/pwa.md [hunk 4]
```diff
@@ -1,25 +1,33 @@
...
+3. Follow your browser's instructions for installing the PWA.
```
Comment (vapurrmaid) [inline environments/pwa.md:19]:
```suggestion
3. Follow your browser's instructions for installing the application as a PWA.
```

File: environments/pwa.md [hunk 5]
```diff
@@ -1,25 +1,33 @@
...
+
+Please note that the code-server app is installed on a per-environment basis.
```
Comment (vapurrmaid) [inline environments/pwa.md:21]:
```suggestion
Please note that applications are currently installed on a per-environment basis.
```

**Review by vapurrmaid — APPROVED** [review]

**Pull request merged**